examples/ecommerce-netlify-functions/functions/addToCart.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { Cart } = require('../models');
+
+const handler = async(event) => {
+  try {
+    if (event.body.cartId) {
+      // get the document containing the specified cartId
+      const cart = await Cart.findOne({ _id: event.body.cartId }).setOptions({ sanitizeFilter: true });
```

Comment:
You also need to test this API manually by sending HTTP requests. This API as written won't work, because you're not running `mongoose.connect()`, so you'll get the "buffering timed out" error